desktop/use-desktop/container.md

```diff
@@ -48,13 +41,31 @@ The **container view** displays **Logs**, **Inspect**, **Terminal**, **Files**,
     - Use the **Clear terminal** icon in the top right-hand corner to clear the logs terminal. 
     - Select and view external links that may be in your logs. 
 
-- Select **Inspect** to view low-level information about the container. You can see the local path, version number of the image, SHA-256, port mapping, and other details.
+### Inspect
+
+- Select **Inspect** to view low-level information about the container. It displays the local path, version number of the image, SHA-256, port mapping, and other details.
+
+### Integrated terminal
+
+From the **Terminal** tab, lets you use the integrated terminal, on a running container, directly within Docker Desktop. You are able to quickly run commands within your container so you can understand its current state or debug when something goes wrong.
```

Comment:
Maybe this is a good area to mention that this tab is labeled **Exec**, I'm assuming to correspond to the command `docker exec` that you describe here.